scripts/tools/fetch-pull-request

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
You can use bash [set builtin](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html#The-Set-Builtin). I will prefer `-x` and `-e`. `-x` prints a trace of command so there wont be any need to write print statements.